environments/magento2.base.yml

```diff
@@ -136,6 +136,39 @@ services:
       - warden
       - default
 
+  crontab:
+    hostname: "${WARDEN_ENV_NAME}-crontab"
+    image: navarr/container-crontab:v0.1.0
```

Comment:
Where can I find the Dockerfile for this image?

Reply:
https://github.com/navarr/container-crontab